yo/generator/server/copyConfigs.js

```diff
@@ -0,0 +1,53 @@
+const boringPackageJSON = require('../../../package.json');
+
+module.exports = async function() {
+
+  const pkgJson = {
+    name: this.options.props.projectName,
+    version: '1.0.0',
+    dependencies: {
+      boringbits: '^' + boringPackageJSON.version,
+    },
+    scripts: {
```

Comment:
So these scripts have been baking over time, at this point they are pretty much a pass through to `npx boring <script>` and serve as convenience.  Since there is a lot, having a way to stuff these scripts into people package.json file makes propigation of these easier.  In theory yeoman does a merge with package.json file making it additive.  So if we ever add a script we simply add it to this file and people will get the update.